src/GraphQL/Types/DateGraphType.cs

```diff
@@ -21,31 +21,31 @@ public override object Serialize(object value)
 
         public override object ParseValue(object value)
         {
-            string inputValue;
-            DateTime dateTime;
-
             if (value is DateTime)
             {
-                inputValue = ((DateTime)value).ToString("yyyy'-'MM'-'dd'T'HH':'mm':'ss.FFFFFFF'Z'");
-            }
-            else
-            {
-                inputValue = value != null ? value.ToString().Trim('"') : string.Empty;
+                return value;
             }
+            
+            string inputValue = value.ToString();
```

Comment:
We will want to handle possible nulls here, so I think just `value?.ToString()` should work.